react-native-cli/index.js

```diff
@@ -203,6 +216,21 @@ function runVerbose(root, projectName) {
   });
 }
 
+function checkNodeVersion() {
+  var packageJson = require(REACT_NATIVE_PACKAGE_JSON_PATH());
+  if (!packageJson.engines || !packageJson.engines.node) {
+    return;
+  }
+  if (!semver.satisfies(process.version, packageJson.engines.node)) {
+    console.error(chalk.red(
+        'You are currently running Node %s but React Native requires %s. ' +
+        'Please use a supported version of Node.'
```

Comment:
Added a link to Getting Started and published react-native-cli 0.1.6. Another change is 'npm start' is now simply an alias for 'react-native start', therefore it will work on Windows (used to call packager.sh).